src/lib/OLED/OLED.cpp

```diff
@@ -14,7 +14,7 @@
  * along with this program. If not, see <http://www.gnu.org/licenses/>.
  */
 
-#if defined(USE_OLED_SPI) || defined(USE_OLED_SPI_SMALL) || defined(USE_OLED_I2C) // This code will not be used if the hardware does not have a OLED display. Maybe a better way to blacklist it in platformio.ini?
+#if defined(USE_OLED_SPI) || defined(USE_OLED_SPI_SMALL) || defined(USE_OLED_I2C) || defined(USE_SW_I2C_OLED) // This code will not be used if the hardware does not have a OLED display. Maybe a better way to blacklist it in platformio.ini?
```

Comment:
`USE_OLED_I2C` is already required so no need to add `USE_SW_I2C_OLED`. 
This same applies to all similar cases.